MonkeyWrench/MonkeyWrench.csproj

```diff
@@ -62,7 +62,6 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="AssemblyInfo.cs" />
```

Comment:
Fixed in https://github.com/mono/monkeywrench/pull/74/commits/7121a27d784dc681fae54903a1ae59c42eb73964